script.js

```diff
@@ -0,0 +1,60 @@
+
+     
+    let pic = document.getElementById('cnvs');
+    let ctx= pic.getContext('2d');
+
+    changeCanvas(pic.width, pic.height)
```

Comment:
also, I think the main purpose of this function is to resize canvas, so resizeCanvas name would be better